components/apimgt/org.wso2.carbon.apimgt.core/src/main/java/org/wso2/carbon/apimgt/core/models/policy/Limit.java

```diff
@@ -18,33 +18,34 @@
 
 package org.wso2.carbon.apimgt.core.models.policy;
 
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+
 /**
  * Contains {@link Limit} based attributes
```

Comment:
I see your point, but there can be classes with same name in different packages in a large codebase. IMHO we should have a link as a best practice, regardless of whether its same class or not.